include/zephyr/sys/mpsc_lockfree.h

```diff
@@ -124,15 +123,15 @@ static inline void rtio_mpsc_init(struct rtio_mpsc *q)
  * @param q Queue to push the node to
  * @param n Node to push into the queue
  */
-static ALWAYS_INLINE void rtio_mpsc_push(struct rtio_mpsc *q, struct rtio_mpsc_node *n)
+static ALWAYS_INLINE void mpsc_push(struct mpsc *q, struct mpsc_node *n)
```

Comment:
push/pop suggests stack. What about `append` and `get` so it corresponds to `slist` API. If i understand correctly it has same functionality as `sys_slist_append` and `sys_slist_get` (with mpsc safeness).

Reply:
The terms and code come directly from https://www.1024cores.net/home/lock-free-algorithms/queues/intrusive-mpsc-node-based-queue as noted in the copyright

Many queues in the wild use push and pop, lifo or fifo, if this naming is a blocking issue I’ll of course spend the time to rename if needed but I’d rather not if I don’t need to.

Reply:
`k_fifo` seems to use `_put()` instead of `_append()`.
https://docs.zephyrproject.org/latest/kernel/services/data_passing/fifos.html#c.k_fifo_put